pkgs/development/python-modules/markupsafe/default.nix

```diff
@@ -6,24 +6,26 @@
 }:
 
 buildPythonPackage rec {
-  pname = "MarkupSafe";
+  pname = "markupsafe";
```

Comment:
https://github.com/pallets/markupsafe/blob/main/setup.cfg#L2

Reply:
What do you mean?

Reply:
For the capitalization of the pname we use the name attribute from the setup.py/cfg metadata AFAIK.

Reply:
No, we normalize it according to PEP 0503: https://nixos.org/manual/nixpkgs/unstable/#contributing

Reply:
Ah okay, I thought that was only for attributes. Thanks!